test/Concurrency/sending_subscript.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-swift-frontend -swift-version 6 %s -emit-silgen | %FileCheck %s
+
+// REQUIRES: concurrency
+
+class NonSendableKlass {}
+
+// CHECK: subscript(_: sending NonSendableKlass) -> sending NonSendableKlass { get }
+
+// CHECK: sil hidden [ossa] @$s17sending_subscript1SVyAA16NonSendableKlassCAEncig : $@convention(method) (@sil_sending @owned NonSendableKlass, S) -> @sil_sending @owned NonSendableKlass {
+struct S {
+  subscript(_: sending NonSendableKlass) -> sending NonSendableKlass { NonSendableKlass() }
```

Comment:
Can you add a test case with multiple results. IIRC there was some sort of code I put in that forced all result types to also need to have sending result as well. The idea is that with time, top level sending result means all results have sending results... but once we support sending on specific results, then we allow for functions not to set the top level sending result flag and vary that specific sending result flag on results. I at least want some more test cases with a tuple result. Let me see if I can find the code in question.

Reply:
Actually, now that I think about it... that property is actually something at the SIL level. Not something at the AST level where we just IIRC represent results with a type instead of using ResultInfo sort of things. Can you just out of an abundance of caution add a subscript with a tuple result test case.

Reply:
@gottesmm 
Thank you for the comment.
I added a test case with a tuple result. https://github.com/swiftlang/swift/pull/79560/commits/98209cdea51213e473f55d1aa6aec56a8ca3ce5b

Reply:
@gottesmm 
Is anything else needed?